packages/digitalocean-js/src/lib/services/image/image.service.ts

```diff
@@ -15,7 +15,9 @@ export class ImageService {
    * ```
    */
   public getAllImages(): Promise<Image[]> {
-    return instance.get(`/images`).then(response => response.data.images);
```

Comment:
Rather than putting a big page size to get all images, I'd like that we instead properly page this method. You can see an example of a couple places we already do this, specifically in this file there is the `getAllDistributionImages` method. That way the user can decide their own page size and page.